vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -30,7 +30,16 @@
     when: EPHEMERAL_CLUSTER == "minikube"
 
   - name: Obtain target cluster kubeconfig
-    shell: "kubectl get secrets {{ CLUSTER_NAME }}-kubeconfig -n {{ NAMESPACE }} -o json | jq -r '.data.value'| base64 -d > /tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
+    k8s_info:
+      kind: secrets
+      name: "{{ CLUSTER_NAME }}-kubeconfig"
+      namespace: "{{ NAMESPACE }}"
+    register: metal3_kubconfig
```

Comment:
I'd prefer if this were named 'kubeconfig' rather than 'kubconfig', but it does seem to be consistently named in your Ansible, so I don't think there is a problem.